web/components/Navbar.tsx

```diff
@@ -122,6 +123,7 @@ export default function Navbar() {
   const router = useRouter();
   const { user, loading, userData, authorized } = useAuth();
   const role = userData?.role;
+  const [currPage, setCurrPage] = useState("FEED");
```

Comment:
You shouldn't have to create new state here - you can get the current route using `router.pathName`